fdbclient/SystemData.cpp

```diff
@@ -632,10 +632,23 @@ std::string encodeExcludedServersKey(AddressExclusion const& addr) {
 	return excludedServersPrefix.toString() + addr.toString();
 }
 
+const KeyRangeRef excludedLocalityKeys(LiteralStringRef("\xff/conf/excludedlocality/"),
```

Comment:
I think current both camel case and snake case naming are used.
We can change this to snake case naming `excluded_locality` thus will be consistent with special keys